src/loadlib.js

```diff
@@ -95,6 +113,17 @@ if (!WEB) {
 
         return true;
     };
+} else {
+    /* TODO: use async/await ? */
+    readable = function(filename) {
+        let xhr = new XMLHttpRequest();
+        /* Following GET request done by searcher_Web will be cached */
+        xhr.open("GET", lua.to_jsstring(filename), false);
```

Comment:
Except in the (probably common) CORS case: the HEAD won't need a preflight, but the GET will (I think?).